### PR TITLE
Release 15.23

### PR DESCRIPTION
**Pull request by gabrielfs7**
Release https://github.com/oat-sa/generis/pull/1030

### CODE REVIEW

File: common/log/class.SingleFileAppender.php [hunk 1]
```diff
@@ -169,7 +169,7 @@ public function doLog(common_log_Item $item)
                 '%t' => $item->getDateTime(),
                 '%r' => $item->getRequest(),
                 '%f' => $item->getCallerFile(),
-                '%g' => implode(',', $item->getTags()),
```
Comment (gabrielfs7) [inline common/log/class.SingleFileAppender.php:172]: - [ ] This is unrelated to the merge. Somebody forgot to release proper generis version

**Review by hectoras — APPROVED** [review]
